config/migrations/20240730174403-mapping-to-variable.sparql

```diff
@@ -0,0 +1,172 @@
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX mobiliteit: <https://data.vlaanderen.be/ns/mobiliteit#>
+PREFIX dct: <http://purl.org/dc/terms/>
+
+
+DELETE {
+    GRAPH ?g {
+        ?s ext:mapping ?o
+    }
+}
+
+INSERT {
+    GRAPH ?g {
+        ?s mobiliteit:variabele ?o
+    }
+}
+
+WHERE {
+    GRAPH ?g {
+        ?s a mobiliteit:Template ;
+        ext:mapping ?o
+    }
+}
+
+;
+
+DELETE {
+    GRAPH ?g {
+        ?s ext:template ?o
+    }
+}
+
+INSERT {
+    GRAPH ?g {
+        ?s mobiliteit:template ?o
+    }
+}
+
+WHERE {
+    GRAPH ?g {
+        ?s a mobiliteit:Template ;
+        ext:template ?o
+    }
+}
```

Comment:
**Question:** Shouldn't the migration be applied to the old type `ext:Template`, like you do on the other predicate migrations? Or do the type migration before migrate the predicate?

Reply:
I probably got confused because in my local setup there are some old predicates saved under mobiliteit:Template. But indeed, you're right. I'll fix it.